ServerCore/Pages/Teams/Create.cshtml

```diff
@@ -0,0 +1,64 @@
+﻿@page "{eventId}/Teams/Create"
+@model ServerCore.Pages.Teams.CreateModel
+
+@{
+    ViewData["Title"] = "Create";
+}
+
+<h2>Create</h2>
+
+<h4>Team</h4>
+<hr />
+<div class="row">
+    <div class="col-md-4">
+        <form method="post">
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+            <div class="form-group">
+                <label asp-for="Team.Name" class="control-label"></label>
+                <input asp-for="Team.Name" class="form-control" />
+                <span asp-validation-for="Team.Name" class="text-danger"></span>
+            </div>
+            <div class="form-group">
+                <label asp-for="Team.Password" class="control-label"></label>
```

Comment:
@asyasky, we're not supposed to have team passwords, right?

Reply:
OK, will remove when the rest of the issues are resolved.